form.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Quiz-App</title>
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="/lib/img/favicon_io/favicon.ico"
+    />
+    <link rel="stylesheet" href="./lib/css/style.css" />
+    <link href="/lib/font/fontawesome/css/fontawesome.css" rel="stylesheet" />
+    <link href="/lib/font/fontawesome/css/brands.css" rel="stylesheet" />
+    <link href="/lib/font/fontawesome/css/solid.css" rel="stylesheet" />
+
+    <script src="./lib/js/bookmark.js" defer></script>
+    <script src="./lib/js/hideAnswer.js" defer></script>
+  </head>
+  <body>
+    <header>
+      <h1>
+        <img src="/lib/img/quiz-gb0d1a774c_640.png" alt="Quiz Apppp ;-)" />
+      </h1>
+    </header>
+    <main class="site__content">
+      <section class="card">
+        <h2>Create a new Question - Card</h2>
+        <form class="card__form" action="">
+          <label class="card__p" for="question">Write your Question!</label>
+          <textarea
+            name="question"
+            id="question"
+            cols="30"
+            rows="10"
+          ></textarea>
+
+          <label class="card__p" for="question">Write your Answer!</label>
+          <textarea
+            name="question"
+            id="question"
+            cols="30"
+            rows="10"
+          ></textarea>
+
+          <label class="card__p" for="tag">Tag:</label>
+          <input type="text" name="tag" />
+
+          <button class="card__button--answer">Submit</button>
```

Comment:
the button touches the bottom menue. maybe more margin-bottom would be good here?
also the button has no type. I think you need a type (i am not 100% sure).